app/policies/synapse_policy.rb

```diff
@@ -7,7 +7,7 @@ def resolve
       return scope.where(permission: visible) unless user
 
```

Comment:
in the topics and synapses policy_scopes, these changes are to fix the circumstances where:
you want to see a topic or synapse which
- is not `commons`
- you did not create
- has a `defer_to_map_id` value which references a map that is not "shared with you" but that you created